EnglishFilter/sections/css_extended.txt

```diff
@@ -1,6 +1,8 @@
 !
 ! Extended CSS rules
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/54102
+theverge.com#$?#.async-hide { opacity: 1.0 !important }
```

Comment:
I don't see this element. How to reproduce it?

Reply:
> 
> 
> I don't see this element. How to reproduce it?

I think the element appears only in a short moment (up to 4 seconds but can be shorter) after each page switches. Look for the inline script tagged as Google Optimize.

Reply:
@Alex-302 Here is the SS. Maybe because my internet is fast, it appears only for about 0.5 sec on average.

![theverge](https://user-images.githubusercontent.com/58900598/80789030-ca798080-8bc5-11ea-9d00-0e0d0c63f0ce.png)